datalad/interface/utils.py

```diff
@@ -450,7 +450,7 @@ def generator_func(*_args, **_kwargs):
                     summary_args = (results, action_summary)
                 else:
                     summary_args = (results,)
-                wrapped_class.custom_result_summary_renderer(*summary_args)
+                wrapped_class.custom_result_summary_renderer(*summary_args, **allkwargs)
```

Comment:
This change breaks the API. I could not find an actual case where this matters (across extensions), but still worth pointing out. The unification with `custom_result_renderer()` makes sense to me in general.

Reply:
Sufficient to point it out in the changelog or do you think this actually needs some sort of "technical" deprecation?

Reply:
Do we have  a test which would go through all `Interface`'s and test that invocation in command line works as expected? if not -- we wouldn't even notice until we break things up.

Also see my comment [above](https://github.com/datalad/datalad/pull/5738/files#r654449041): I think they should be passed as 

```suggestion
                wrapped_class.custom_result_summary_renderer(*summary_args, call_kwargs=allkwargs)
```
here.

Also, in principle, we could use then `inspect` (and cache the check) if `custom_result_summary_renderer` expects `call_kwargs` or has a swallow all `**` (if that is possible to check -- didn't look): if not --  issue a DeprecationWarning to alert that interface is changing.